rules-tests/DeadCode/Rector/Property/RemoveUnusedPrivatePropertyRector/Fixture/constructor_with_middle_params.php.inc

```diff
@@ -29,7 +29,7 @@ namespace Rector\Tests\DeadCode\Rector\Property\RemoveUnusedPrivatePropertyRecto
 
 final class ConstructorWithMiddleParams
 {
-    public function __construct(int $contentFinder, \stdClass $stdClass)
+    public function __construct(\stdClass $stdClass)
```

Comment:
The first param removal should be reverted, as that BC break on the caller, removal should only happen when next param is also removed. 

This fixture is for middle param that can't be removed.

I will look into it without node remover :)

Reply:
https://github.com/rectorphp/rector-src/pull/4016